src/test/py/bazel/py_test.py

```diff
@@ -46,6 +46,9 @@ def createSimpleFiles(self):
         ])
 
   def testSmoke(self):
+    if test_base.TestBase.IsDarwin():
```

Comment:
Looks like `IsDarwin` doesn't exist in 6.x. Can you replace it with `sys.platform == 'darwin'`?

Reply:
You'll also need to `import sys` first.

Reply:
Done.

cc: @meteorcloudy @Wyverald

Reply:
Can you please also disable the failing `//scripts/docs:rewriter_test` from both presubmit.yml and postsubmit.yml for macOS to fix the presubmit?

Reply:
@meteorcloudy here's the PR: https://github.com/bazelbuild/bazel/pull/20758